src/VisionModel/vgg.jl

```diff
@@ -60,32 +53,37 @@ function classifier_layers(imsize, nclasses, fcsize, dropout)
   return layers
 end
 
-function vgg(imsize; config, batchnorm=false, inchannels=3, nclasses, fcsize=4096, dropout=0.5)
-  conv = convolutional_layers(config, batchnorm, inchannels)
-  class = classifier_layers(imsize, nclasses, fcsize, dropout)
+function vgg(imsize; config, batchnorm=false, nclasses)
+  conv = convolutional_layers(config, batchnorm, 3)
+  class = classifier_layers(imsize, nclasses, 4096, 0.5)
```

Comment:
I would prefer to revert back to using `fcsize` and `dropout` as parameters that are passed in. This allows someone to use the `vgg` function to construct a network that isn't one of the standard configurations.

Reply:
Shall I change the value of inplanes i.e. 3 to inplanes?